### PR TITLE
Breadcrumbs added

### PR DESCRIPTION
**Pull request by basil7vals**

### CODE REVIEW

**Comment by basil7vals** [comment]
Cycle - 0 completed

File: app/views/projects/_project.html.erb [hunk 1]
```diff
@@ -2,9 +2,8 @@
 <% if can?(:index, project) %>
 <div class="card">
   <div class="card-body">
-    <h5 class="card-title"><%= link_to project.title, project%>
-      <span class="badge badge-secondary badge-pill float-right ml-2"><%= project.issues.size%></span>
-      <span class="badge badge-danger badge-pill float-right"><%= link_to 'Issues', '#', class: 'text-white'%></span>
+    <h5><%= link_to project.title, project, style: "color: #002f6c;"%>
+      <span class="badge badge-white badge-pill-sm border float-right ml-2"  style="border-color: #ebccd1 !important; background-color: #f2dede !important; text-color: #a94442; color: #a94442;"><%= link_to project.issues.size, project,class: "font-weight-bold", style: "color: #a94442;"%></span>
```
Comment (abubakarazeem) [inline app/views/projects/_project.html.erb]: define CSS class

**Comment by abubakarazeem** [comment]
review cycle 1

File: app/views/projects/_project.html.erb (hunk 1 above)
Comment (basil7vals) [reply app/views/projects/_project.html.erb]: Already updated in 

> projects_ui branch

**Pull request merged**